suite-native/device/src/components/UninitializedDeviceModalAppendix.tsx

```diff
@@ -0,0 +1,88 @@
+import { VStack, HStack, Text, Box } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Icon, IconName } from '@suite-common/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+import { TxKeyPath } from '@suite-native/intl';
+
+const listItemStyle = prepareNativeStyle(() => ({
+    width: '90%',
+}));
+
+const iconBackgroundStyle = prepareNativeStyle<{ color: Color }>((utils, { color }) => ({
+    backgroundColor: utils.colors[color],
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: utils.spacings.extraLarge,
+    height: utils.spacings.extraLarge,
+    borderWidth: color ? utils.borders.widths.small : 0,
+    borderColor: utils.colors.borderElevation1,
+    borderRadius: 12,
+}));
+
+type ListItemProps = {
+    iconBackgroundColor?: Color;
+    translationKey: TxKeyPath;
+    iconColor?: Color;
+} & (
+    | {
+          iconName: IconName;
+          iconNumber?: never;
+      }
+    | {
+          iconName?: never;
+          iconNumber: number;
+      }
+);
+
+const ListItem = ({
+    iconName,
+    iconColor,
+    iconNumber,
+    iconBackgroundColor = 'backgroundTertiaryDefaultOnElevation1',
+    translationKey,
+}: ListItemProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <HStack spacing={12} alignItems="center">
+            <Box style={applyStyle(iconBackgroundStyle, { color: iconBackgroundColor })}>
+                {iconNumber && <Text color={iconColor}>{iconNumber}</Text>}
+                {!iconNumber && iconName && (
+                    <Icon name={iconName} color={iconColor} size="mediumLarge" />
+                )}
+            </Box>
+            <Text style={applyStyle(listItemStyle)}>
+                <Translation id={translationKey} />
+            </Text>
+        </HStack>
+    );
+};
+
+export const UninitializedDeviceModalAppendix = () => {
+    return (
+        <VStack>
+            <Text variant="callout">
+                <Translation id="moduleDevice.noSeedModal.appendix.title" />
+            </Text>
+            <VStack spacing="medium" paddingTop="large">
+                <ListItem
+                    iconNumber={1}
+                    translationKey="moduleDevice.noSeedModal.appendix.lines.1"
```

Comment:
Should not be here a number instead of a icon? Same in the next item.

Reply:
Now it diverges from the design since the check icons are in every item.
![Screenshot 2024-07-02 at 7 59 00](https://github.com/trezor/trezor-suite/assets/26143964/1cf72e2a-24f7-4ca6-b7aa-8a2637f80162)